docker/coturn/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM alpine:3.11
+
+RUN apk add --no-cache coturn
+
+CMD ["turnserver", "--prod", "--fingerprint", "--use-auth-secret", "--lt-cred-mech", "--static-auth-secret=$STATIC_SECRET", "--realm=$REALM", "--no-multicast-peers", "--no-tls", "--no-dtls"]
```

Comment:
Hello
I have the problem, that the variables $STATIC_SECRET and $REALM used in CMD are not recognized here.....When I set the hardcoded values here instead of the variables it works.....
What could be the problem here?